MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -8,6 +8,8 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Net.Http;
```

Comment:
Try to maintain the ordering. That is, System.Net.Http should come before System.Threading.

Reply:
Okay! I will put them in alphabetical order.

Reply:
FYI you can use the "Remove and Sort Usings" content menu command to do this.

Reply:
@mhutch I was manually putting them all in alphabetical order yesterday!! This is so helpful :D